util/lambda/index.js

```diff
@@ -0,0 +1,244 @@
+/* eslint-disable no-useless-catch */
+/* eslint-disable no-unused-vars */
+const JSZip = require("jszip");
+const fs = require("fs");
+const AWS = require("aws-sdk");
+const logger = require("../../logger");
+
+const lambda = new AWS.Lambda({
+  apiVersion: "2015-03-31",
+  region: process.env.AWS_REGION || "",
+  accessKeyId: process.env.AWS_ACCESS_KEY_ID || "",
```

Comment:
we should use role base auth as this internal Lambda invocation

Reply:
working on it

Reply:
Infra team told it would take time. Going forward with aws creds for now

Reply:
@lokey this will be backlog in FTR.

Reply:
Tickets to follow up:
https://www.notion.so/rudderstacks/Assign-lambda-required-role-policies-to-transformer-pods-e4f6d0fa7e1749d9a8945809b4addea2
https://www.notion.so/rudderstacks/Assign-lambda-required-role-policies-to-transformer-pods-b9b5d83b8e6048389960dbc3b988d305